### DISCUSSION TITLE
Loudness normalization

### DISCUSSION THREAD

**Discussion by Anlaoui** [fast4x/RiMusic#1555] [General]
Hello and first thanks a lot for your fantastic app!
Bit of a nerd question here: I was wondering how is the audio normalized when Loudness nomalization is on. Does it turn up quiet songs or only turns down loud ones? What's the reference level? (for example, Spotify doesn't have the same reference level as Deezer,, or Tidal, etc.)

**Comment by fast4x** [comment]
only turns down loud ones...

**Comment by Anlaoui** [comment]
Okay nice!Message ID: 
***@***.***>
